Makefile

```diff
@@ -204,3 +204,44 @@ migrate: migrate-lms migrate-cms
 # Part of https://github.com/openedx/wg-developer-experience/issues/136
 ubuntu-requirements: ## Install ubuntu 22.04 system packages needed for `pip install` to work on ubuntu.
 	sudo apt install libmysqlclient-dev libxmlsec1-dev
+
+eslint:	## check javascript for quality issues
+	python scripts/eslint.py
+
+xsslint: ## check xss for quality issuest
+	python scripts/xsslint/xss_linter.py \
+	--rule-totals \
+	--config=scripts.xsslint_config \
+	--thresholds=scripts/xsslint_thresholds.json
```

Comment:
In my testing, this correctly flags xss violations which are over the threshold 👍🏻 

However, I don't see how to drill into the results of the violations. The script printed here doesn't exist in edx-platform:
```
FAILURE: XSSLinter Failed.

Too many javascript-escape violations (2).
The javascript-escape limit is 1.
run the following command to hone in on the problem:
  ./scripts/xss-commit-linter.sh -h
```
```
app@812737e89236:~/edx-platform$ ./scripts/xss-commit-linter.sh -h
bash: ./scripts/xss-commit-linter.sh: No such file or directory
```
Was that an existing problem with xsslint? If so, we can ignore the problem for now, and fix later.

Reply:
@kdmccormick `/scripts/xss-commit-linter.sh` file does not exist. I checked, and this is an existing problem. Should I remove the script name from the error message, or can we fix this later?

Reply:
Let's leave it as-is and fix later.